<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HpiViewWidget</class>
 <widget class="QWidget" name="HpiViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HPI Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="m_gridLayout_main">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="1" rowspan="2">
      <widget class="QGroupBox" name="m_groupBox_3dControl">
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>3D Control</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="m_groupBox_loadDigitizers">
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>1 - Load digitizers</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Number of loaded HPI coils:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="m_label_numberLoadedCoils">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Number of loaded additional digitizers:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="m_label_numberLoadedDigitizers">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Number of loaded fiducials:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="m_label_numberLoadedFiducials">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Number of loaded EEG:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="m_label_numberLoadedEEG">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLineEdit" name="m_lineEdit_filePath"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="m_pushButton_loadDigitizers">
          <property name="text">
           <string>Load Digitizers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="5">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="m_groupBox_hpiFitting">
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>2 - HPI fitting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="10" column="0" colspan="4">
         <widget class="QToolButton" name="m_pushButton_doSingleFit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Perform single HPI fit</string>
          </property>
         </widget>
        </item>
        <item row="17" column="2">
         <widget class="QLabel" name="m_label_gofCoil3Description">
          <property name="text">
           <string>Coil 3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QLabel" name="m_label_mat20">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="16" column="0" colspan="4">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Fitting errors:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="m_spinBox_freqCoil3">
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>190</number>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="m_label_gofCoil2Description">
          <property name="text">
           <string>Coil 2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="23" column="3">
         <widget class="QLabel" name="m_label_mat03">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="m_label_freqCoil4">
          <property name="text">
           <string>Coil 4:</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="m_label_gofCoil1Description">
          <property name="text">
           <string>Coil 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="25" column="2">
         <widget class="QLabel" name="m_label_mat22">
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <widget class="QLabel" name="m_label_mat30">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Coil frequencies (as set in LabView):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="25" column="3">
         <widget class="QLabel" name="m_label_mat23">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="26" column="2">
         <widget class="QLabel" name="m_label_mat32">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QSpinBox" name="m_spinBox_freqCoil2">
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>165</number>
          </property>
         </widget>
        </item>
        <item row="26" column="1">
         <widget class="QLabel" name="m_label_mat31">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="23" column="2">
         <widget class="QLabel" name="m_label_mat02">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="m_label_freqCoil1">
          <property name="text">
           <string>Coil 1:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QSpinBox" name="m_spinBox_freqCoil4">
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>220</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="m_label_freqCoil3">
          <property name="text">
           <string>Coil 3:</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <widget class="QLabel" name="m_label_mat10">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="24" column="2">
         <widget class="QLabel" name="m_label_mat12">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="m_spinBox_freqCoil1">
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>155</number>
          </property>
         </widget>
        </item>
        <item row="24" column="1">
         <widget class="QLabel" name="m_label_mat11">
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="24" column="3">
         <widget class="QLabel" name="m_label_mat13">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="26" column="3">
         <widget class="QLabel" name="m_label_mat33">
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="4">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="20" column="0" colspan="4">
         <widget class="QLabel" name="m_label_fitFeedback">
          <property name="text">
           <string>No fit yet</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="4">
         <widget class="QCheckBox" name="m_checkBox_continousHPI">
          <property name="text">
           <string>Do continous HPI fitting</string>
          </property>
         </widget>
        </item>
        <item row="22" column="0" colspan="4">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Computed dev/head transformation matrix:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="25" column="1">
         <widget class="QLabel" name="m_label_mat21">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="17" column="3">
         <widget class="QLabel" name="m_label_gofCoil4Description">
          <property name="text">
           <string>Coil 4</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="m_label_gofCoil1">
          <property name="text">
           <string>0mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="m_label_freqCoil2">
          <property name="text">
           <string>Coil 2:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="3">
         <widget class="QLabel" name="m_label_gofCoil4">
          <property name="text">
           <string>0mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="18" column="2">
         <widget class="QLabel" name="m_label_gofCoil3">
          <property name="text">
           <string>0mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="21" column="0" colspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="23" column="1">
         <widget class="QLabel" name="m_label_mat01">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLabel" name="m_label_gofCoil2">
          <property name="text">
           <string>0mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QLabel" name="m_label_mat00">
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Average:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QLabel" name="m_label_averagedFitError">
          <property name="text">
           <string>0mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="19" column="3">
         <widget class="QDoubleSpinBox" name="m_doubleSpinBox_maxHPIContinousDist">
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="19" column="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Allowed:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QCheckBox" name="m_checkBox_useComp">
          <property name="text">
           <string>Use compensators</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="m_checkBox_useSSP">
          <property name="text">
           <string>Use SSP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
